4.0/HowToBuild.md

```diff
@@ -0,0 +1,22 @@
+# How to create the document
+1. Install the Docker Daemon on your computer (Google is your friend)
+2. If running WSL or WSL2 make sure that you can talk to the Docker Daemon from the console (again, Google this)
+3. Run this command to create the container:
+
+```
+docker image build --tag documentBuilder -f docker/Dockerfile .
```

Comment:
Docker tags have two parts, a repository (with optional org), and then the tag, separated by a colon. For example: `asvs/documentbuilder:version1.0`. One of the restrictions is that the repository portion of the tag must be lowercase. More information is available [on the docker docs here](https://docs.docker.com/engine/reference/commandline/tag/#extended-description).

```suggestion
docker image build --tag asvs/documentbuilder:latest -f docker/Dockerfile .
```